helm-charts/templates/postgres/initdb-conf.yaml

```diff
@@ -403,5 +403,23 @@ data:
           PRIMARY KEY (CONSUMER_KEY)
         );    
         -- End of Non Prod IDP table --
+
+        -- Insert Backoffice Internal Data --
```

Comment:
Why we need this?

Reply:
Since the internal backoffice API was not exposed, the raw SQL statements were added to create an API in the database for testing.

Reply:
lets remove this, we dont need this